back/spec/acceptance/initiative_status_changes_spec.rb

```diff
@@ -95,12 +97,33 @@
         end
       end
 
-      describe do
-        let(:body_multiloc) { nil }
-        let(:author_multiloc) { nil }
+      context 'when the status transition requires official feedback' do
+        describe do
+          let(:body_multiloc) { nil }
+          let(:author_multiloc) { nil }
 
-        example_request '[error] Create a status change on an initiative without feedback' do
-          assert_status 422
+          example_request '[error] Create a status change on an initiative without feedback' do
+            assert_status 422
+          end
+        end
+      end
+
+      context 'when the status transition does not require official feedback' do
+        before do
+          create(
+            :initiative_status_change,
+            initiative: @initiative, initiative_status: @status_approval_pending
+          )
+        end
+
+        describe do
+          let(:initiative_status_id) { @status_proposed.id }
+          let(:body_multiloc) { nil }
+          let(:author_multiloc) { nil }
+
+          example_request 'Create a status change on an initiative without feedback' do
+            assert_status 201
+          end
```

Comment:
I am currently experimenting with not requiring official feedback for the transition `approval_required` -> `proposed`.

I actually think this is a bad idea, as different platforms may have very different ideas around what to say/not say in official feedbacks, but at least I know how to do it if we decide it has some value.

We can discuss tomorrow (Wednesday).